src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/Microsoft.AspNetCore.Components.Razor.Extensions.csproj

```diff
@@ -6,6 +6,8 @@
     <RootNamespace>Microsoft.AspNetCore.Components.Razor</RootNamespace>
     <Description>Extensions to the Razor compiler to support building Razor Components.</Description>
     <IsProductPackage>true</IsProductPackage>
+    <!-- Excluded because this assembly has a 3rd party dependency. -->
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
Will the various exclusions cause problems when we get to source-building AspNetCore i.e. will the AspNetCore dependency graph have gaps?

Reply:
huh. I'm not super sure why this project still exists.

Reply:
In this specific case, I believe it's okay to exclude. We're only trying to source-build the shared framework and the Razor SDK. As long as this isn't used in either place, it doesn't need to support source build.

But if I'm wrong, then we need to either support a 3rd party dependency in source build or remove the dependency on AngleSharp